src/Preface.lidr

```diff
@@ -314,24 +314,32 @@ paths through the material can be found in the file `deps.html`.
 
 === System Requirements
 
-\todo[inline]{Rewrite this for Idris}
+Idris runs on Windows, Linux, and macOS.  You will need:
 
-Coq runs on Windows, Linux, and OS X.  You will need:
+  - A current installation of Idris, available from
+    \href{https://www.idris-lang.org/}{the Idris home page}. Everything should
+    work with version 1.0. (Version 1.1.0 should work, but hasn't been tested by
```

Comment:
Actually it does work if you delete the overlapping instances I've pushed to prelude :) I plan to do a PR for that soon.